src/SuperBlog/Controller/ArticleController.php

```diff
@@ -23,8 +26,15 @@ public function __construct(ArticleRepository $repository, Twig_Environment $twi
         $this->twig = $twig;
     }
 
-    public function show($id)
+    /**
+     * @param $id
+     * @throws LoaderError
+     * @throws RuntimeError
+     * @throws SyntaxError
+     */
+    public function show($id): void
     {
+        $id = (int) $id;
```

Comment:
The project seemed to be using a string ID, I just want to be sure the change makes sense.